sky/packages/updater/lib/main.dart

```diff
@@ -67,14 +100,15 @@ class UpdateTask {
   String _dataDir;
   String _tempPath;
 
-  _readLocalManifest() async {
-    String manifestPath = path.join(_dataDir, kManifestFile);
-    String manifestData = await new File(manifestPath).readAsString();
-    _currentManifest = yaml.loadYaml(manifestData, sourceUrl: manifestPath);
+  Future _readLocalManifest() async {
+    String bundlePath = path.join(_dataDir, kBundleFile);
```

Comment:
It's not really a danger at all for the people on this project. (It does have a minor documentary benefit, in that it's easier to tell what variables are meant to be changed and which aren't.)

We've been moving more towards using "final" everywhere we can.

This is just a style nit. I'll leave it up to you. :-)